src/epd2in13bc/graphics.rs

```diff
@@ -31,7 +31,7 @@ impl DrawTarget for Display2in13bc {
         I: IntoIterator<Item = Pixel<Self::Color>>,
     {
         for pixel in pixels {
-            self.draw_helper_tri(WIDTH, HEIGHT, pixel)?;
+            self.draw_helper_tri(WIDTH, HEIGHT, pixel, DisplayColorRendering::Positive)?;
```

Comment:
Unfortunately i do not have 2.13" available right now, but this change should preserve existing behaviour.

I can order a 2.13" screen (as i may need it for the other project) and test this change, but it will take some time, at least couple of months

Reply:
No, I just wanted to confirm my thoughs of the change in that line :+1: